packages/vaadin-lumo-styles/src/components/side-nav-item.css

```diff
@@ -167,4 +167,17 @@
     color: var(--vaadin-selection-color-text, var(--lumo-primary-text-color));
     border-radius: var(--lumo-border-radius-m);
   }
+
+  .sr-only {
```

Comment:
In `vaadin-select` we inject the style within the component itself:
https://github.com/vaadin/web-components/blob/4b316158a4a4f702f032bc9940fc82f0faa840f4/packages/select/src/vaadin-select.js#L140-L142

Would that work as an alternative to duplicating this?

Reply:
It's added in base styles, but in Lumo most of components don't use base styles and remove them unless `includeBaseStyles: true` is set. This means we have to duplicate lots of CSS. I'll try improving it separately.